mission_control/navigator_launch/launch/perception.launch

```diff
@@ -4,4 +4,5 @@
   <!--node name="start_gate_vision" pkg="navigator_controller" type="start_gate_manual.py" respawn="True"/-->
 
   <node name="scan_the_code" pkg="navigator_vision" type="model_detector.py" respawn="True"/>
+  <node name="pinger_finder" pkg="navigator_vision" type="pinger_finder.py" />
```

Comment:
This is supposed to go in the sonar package, which is currently a dependency from Sub8.
It is definitely perception but currently has no place in our file structure.